### ISSUE TITLE
Error from python syntax in 2.6, works in 2.7

### ISSUE THREAD

**Issue by cmdcolin** [combogenomics/medusa#19]
Hi there

I was running the program and got an error at this stage

```
Building the network...  File "medusa_scripts/netcon_mummer.py", line 170
    G[n1][n2]['orientation_max']=list({tuple(i) for i in G[n1][n2]['orientation'] if G[n1][n2]['orientation'].count(i)==max_count})
```

Under python2 .6 it is a syntax error but works in 2.7

I just wanted to prove it to myself and made this small code which fails under 2.6 but works in 2.7

```
>>> r=[(1,1),(2,2),(3,3)]
>>> list({tuple(i) for i in r })
```

so in effect could be worth making the README say python2 .7 instead of 2.6 :)

**Comment by EBosi** [comment]
Yes, dict comprehension doesn't work in 2.6! Will fix asap, for now use 2.7+

Il 16 giu 2017 17:58, "Colin Diesh" <notifications@github.com> ha scritto:

**Comment by cmdcolin** [comment]
Many thanks. Dont necessarily need 2.6 support, just something I noticed.

At the moment, I have run into some different errors but I will make a new issue

Thanks!
